.github/workflows/deploy.yml

```
@@ -251,21 +272,41 @@ jobs:
\ No newline at end of file
+          if [ "${{ inputs.environment }}" = "ephemeral" ]; then
+            ENV_NAME="pr-${{ inputs.pr_number }}"
+          else
+            ENV_NAME="${{ inputs.environment }}"
+          fi
+          echo ${{ secrets.VITE_CLERK_PUBLISHABLE_KEY }} | bunx wrangler secret put --env $ENV_NAME VITE_CLERK_PUBLISHABLE_KEY
+          echo ${{ secrets.VITE_CONVEX_URL }} | bunx wrangler secret put --env $ENV_NAME VITE_CONVEX_URL
+          echo ${{ secrets.CLERK_SECRET_KEY }} | bunx wrangler secret put --env $ENV_NAME CLERK_SECRET_KEY
+          echo ${{ secrets.VITE_POSTHOG_API_KEY }} | bunx wrangler secret put --env $ENV_NAME VITE_POSTHOG_API_KEY
+          echo ${{ secrets.VITE_POSTHOG_API_HOST }} | bunx wrangler secret put --env $ENV_NAME VITE_POSTHOG_API_HOST
+          echo ${{ secrets.VITE_POSTHOG_PROJECT_ID }} | bunx wrangler secret put --env $ENV_NAME VITE_POSTHOG_PROJECT_ID
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)